app/views/govuk_component/docs/title.yml

```diff
@@ -27,3 +27,10 @@ examples:
       context: Publication
       title: My page title which is often really long and verbose and has lots of extra words it doesn't need
       average_title_length: long
+  title_on_dark_background:
```

Comment:
We could call this `html_publication_title`, or use the description to indicate where this is used.

Reply:
Description seems like a good shout 👍